step-capstone/src/__tests__/GetScoreTests.test.js

```diff
@@ -0,0 +1,74 @@
+import '@testing-library/jest-dom'
+import {getScore} from '../scripts/Suggestion'
+
+
+/*
+ * Score testing
+ */
+
+const ZERO_PROMINENCE = { total: 1, index: 1 }
+
+test("Handles category score correctly", () => {
```

Comment:
Nit: These tests are not super descriptive, what is correct in this context?

Reply:
This is checking the general use case of getScore, where some categories match. changed description for clarity

Reply:
I suggest "check getScore() function on a usual case".